service/cmd/keycloak_data.yaml

```
@@ -71,6 +71,16 @@ realms:
           secret: secret
           protocolMappers:
             - *customAudMapper
+      - client:
+          clientID: cli-client
+          enabled: true
+          name: cli-client
+          serviceAccountsEnabled: false
+          publicClient: true
+          redirectUris:
+            - 'http://localhost:*/*'
```

I think we've encountered issues before with `http://localhost:*/*`.
```suggestion
            - 'http://localhost:*'
```

---

done